src/core/core.js

```diff
@@ -1,6 +1,6 @@
 /** @module Core */
 import { provideCore } from '@yext/answers-core/lib/commonjs';
-
+import {uuid} from './utils/uuid';
```

Comment:
Stylistic comment: there should be a space between what's imported and the curly braces: `{ uuid }`